docs/community/topics/i18n.rst

```diff
@@ -1,120 +1,118 @@
 Internationalization
 ====================
 
-.. warning::
+We use `Gettext <https://docs.python.org/3/library/gettext.html>`__ message catalogs for the theme's internationalization (I18N) and localization (L10N) and `pybabel <https://babel.pocoo.org/en/latest/messages.html>`__ to work with them.
 
-   This theme is still in the process of setting up internationalization.
-   Some of the text below may not yet be correct (for example, we do not yet have a ``locales/`` directory).
-   Follow these issues to track progress:
+User-facing strings in the theme should be internationalized, so that they can be translated into other languages.
+This section covers how to mark text as translatable and how to translate the theme.
 
-   - https://github.com/pydata/pydata-sphinx-theme/issues/1162
-   - https://github.com/pydata/pydata-sphinx-theme/issues/257
+..Note
+   Internationalization (or i18n) is the process of marking strings for translation so that the strings can be extracted from the source code and given to translators. Localization (l10n) is the process of translating the marked strings into different languages.
```

Comment:
Funny enough localisation is also used to refer to what you mentioned: dates, currency, etc. 
Depending on the translation context I suppose it will or not include items like this.

Reply:
I remember it because localisation is not only about language, but other things. Like fr_fr, and fr_ca will have different words and units, even if both are french. 

It's also coherent with the usage of the term [LOCALE](https://en.wikipedia.org/wiki/Locale_(computer_software)), 

> In [computing](https://en.wikipedia.org/wiki/Computing), a locale is a set of [parameters](https://en.wikipedia.org/wiki/Parameter) that defines the user's language, region and any special variant preferences that the user wants to see in their [user interface](https://en.wikipedia.org/wiki/User_interface). Usually a locale identifier consists of at least a language code and a country/region code.


But yes, that's the terms of i18n and l10n (though I find the 18 and 10 horrible, and I think it should have been fully spelled, but not my decision).

Reply:
I tend to agree. I'm not a big fan of numeronyms. But I do make an exception for i18n because 18 starts to be quite a lot. I cannot seem to type internationalization without making a mistake, and if English were my second language I bet it would also be a hard word to say, whereas localization doesn't trip me up.